### ISSUE TITLE
Evohome TRV temp's flatlined since v3.5135

### ISSUE THREAD

**Issue by stuiow** [domoticz/domoticz#661]
Actual temperatures of each Evohome temperature have flatlined since installing beta 3.5135.

The hot water is unaffected.

**Comment by gizmocuz** [comment]
Maybe nice to mention to the developer, from which version ?

**Comment by stuiow** [comment]
beta 3.5135?

It was OK on the beta release before that.

**Comment by gizmocuz** [comment]
Not the answer i was expecting... let me rephrase that differently

What beta version did you used before you upgraded to 3.5135 ?

Also, did you check in the commit log here on git, what has been changed between those versions, and did you check if those changes had 'anything' to do with evohome ?

**Comment by stuiow** [comment]
I believe it was 3.5114

**Comment by gizmocuz** [comment]
https://github.com/domoticz/domoticz/commits/master

**Comment by gizmocuz** [comment]
Did you try restarting ? I dont think there are evohome changes

**Comment by stuiow** [comment]
The same problem occurred a few weeks ago but was sorted in another beta (bug is was reported on here a few posts down).
I've tried rebooting domoticz/pi2 but it doesn't have any affect.
I'll try rebooting evohome next.

**Comment by stuiow** [comment]
Tried restarting both evohome and domoticz, still have current temperatures static.
Setpoints and DHW still appear to be working.
I'm now currently on Beta 3.5143.
https://github.com/domoticz/domoticz/issues/635 was the previous issue, same symptoms, but 3.5104 fixed whatever it was.
Also, nothing recent on commits jumps out at me to involve evohome.

**Comment by stuiow** [comment]
Done some playing around this morning, using a new raspberry image and updating straight to beta  3.5143.
Still same result.
Setpoints are still getting through to domoticz. I noticed that there is a 1hr difference in setpoints again. Maybe this is causing an issue?
For example, i set a room to 17.5c until 11:35. That came through to Domoticz about 3/4 seconds later, right temperature but until 12:35.

**Comment by Evsdd** [comment]
I'm taking a look at this at the moment. The May 1st evohome changes shouldn't have impacted the temperature updating.  I see you've posted on the forum and I'll reply there too with a few questions about your specific evohome hardware and see if I can help troubleshoot.

**Comment by stuiow** [comment]
Thanks Dan. 
I'll do what you've asked for later when I get home. Thanks for looking into this.
Be in touch later tonight.

**Comment by stuiow** [comment]
Right, problem solved.
There were for whatever reason, multiple "Evotouch colour" devices in my devices list within Domoticz.
Deleting all the old ones and leaving the current one, then rebooting Domoticz brought everything back working.

Thanks all for your assistance!
